api/estados/saopaulo.js

```diff
@@ -1,652 +1,654 @@
+    'Neves Paulista',
+    'Nhandeara',
+    'Nipoã',
+    'Nova Aliança',
+    'Nova Campina',
+    'Nova Canaã Paulista',
+    'Nova Castilho',
+    'Nova Europa',
+    'Nova Granada',
+    'Nova Guataporanga',
+    'Nova Independência',
+    'Nova Luzitânia',
+    'Nova Odessa',
+    'Novais',
+    'Novo Horizonte',
+    'Nuporanga',
+    'Ocauçu',
+    'Óleo',
+    'Olímpia',
+    'Onda Verde',
+    'Oriente',
+    'Orindiúva',
+    'Orlândia',
+    'Osasco',
+    'Oscar Bressane',
+    'Osvaldo Cruz',
+    'Ourinhos',
+    'Ouro Verde',
+    'Ouroeste',
+    'Pacaembu',
+    'Palestina',
+    'Palmares Paulista',
+    "Palmeira d'Oeste",
+    'Palmital',
+    'Panorama',
+    'Paraguaçu Paulista',
+    'Paraibuna',
+    'Paraíso',
+    'Paranapanema',
+    'Paranapuã',
+    'Parapuã',
+    'Pardinho',
+    'Pariquera-Açu',
+    'Parisi',
+    'Patrocínio Paulista',
+    'Paulicéia',
+    'Paulínia',
+    'Paulistânia',
+    'Paulo de Faria',
+    'Pederneiras',
+    'Pedra Bela',
+    'Pedranópolis',
+    'Pedregulho',
+    'Pedreira',
+    'Pedrinhas Paulista',
+    'Pedro de Toledo',
+    'Penápolis',
+    'Pereira Barreto',
+    'Pereiras',
+    'Peruíbe',
+    'Piacatu',
+    'Piedade',
+    'Pilar do Sul',
+    'Pindamonhangaba',
+    'Pindorama',
+    'Pinhalzinho',
+    'Piquerobi',
+    'Piquete',
+    'Piracaia',
+    'Piracicaba',
+    'Piraju',
+    'Pirajuí',
+    'Pirangi',
+    'Pirapora do Bom Jesus',
+    'Pirapozinho',
+    'Pirassununga',
+    'Piratininga',
+    'Pitangueiras',
+    'Planalto',
+    'Platina',
+    'Poá',
+    'Poloni',
+    'Pompéia',
+    'Pongaí',
+    'Pontal',
+    'Pontalinda',
+    'Pontes Gestal',
+    'Populina',
+    'Porangaba',
+    'Porto Feliz',
+    'Porto Ferreira',
+    'Potim',
+    'Potirendaba',
+    'Pracinha',
+    'Pradópolis',
+    'Praia Grande',
+    'Pratânia',
+    'Presidente Alves',
+    'Presidente Bernardes',
+    'Presidente Epitácio',
+    'Presidente Prudente',
+    'Presidente Venceslau',
+    'Promissão',
+    'Quadra',
+    'Quatá',
+    'Queiroz',
+    'Queluz',
+    'Quintana',
+    'Rafard',
+    'Rancharia',
+    'Redenção da Serra',
+    'Regente Feijó',
+    'Reginópolis',
+    'Registro',
+    'Restinga',
+    'Ribeira',
+    'Ribeirão Bonito',
+    'Ribeirão Branco',
+    'Ribeirão Corrente',
+    'Ribeirão do Sul',
+    'Ribeirão dos Índios',
+    'Ribeirão Grande',
+    'Ribeirão Pires',
+    'Ribeirão Preto',
+    'Rifaina',
+    'Rincão',
+    'Rinópolis',
+    'Rio Claro',
+    'Rio das Pedras',
+    'Rio Grande da Serra',
+    'Riolândia',
+    'Riversul',
+    'Rosana',
+    'Roseira',
+    'Rubiácea',
+    'Rubinéia',
+    'Sabino',
+    'Sagres',
+    'Sales',
+    'Sales Oliveira',
+    'Salesópolis',
+    'Salmourão',
+    'Saltinho',
+    'Salto',
+    'Salto de Pirapora',
+    'Salto Grande',
+    'Sandovalina',
+    'Santa Adélia',
+    'Santa Albertina',
+    "Santa Bárbara d'Oeste",
+    'Santa Branca',
+    "Santa Clara d'Oeste",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)